test/regtest.py

```diff
@@ -186,12 +207,17 @@ def main():
                       help="sets the logging level (DEBUG, INFO, WARNING)")
   parser.add_argument("--output-log", action="store", dest="log_path", type=str,
                       help="sets the output log path. (std out by default)")
+  parser.add_argument("--languages", action="store", default="c",
```

Comment:
This should be implemented using `choices`: https://docs.python.org/2/library/argparse.html#choices
Then you don't have to do the manual checking of this argument above.

Reply:
See this one for how to do multiple choices: https://stackoverflow.com/questions/44342813/multiple-choices-command-line-argument